addons/sairam.quadtree/QuadTree.gd

```diff
@@ -48,35 +53,42 @@ func add_body(body: VisualInstance) -> Object:
 		# subdivide if necessary
 		_subdivide()
 		# update the body's quadtree.
-		return update_body(body)
+#		return update_body(body, false)
```

Comment:
There was an issue so I just disabled this line.

Reply:
Let me look. Without that, it would hang for me.

Reply:
Well, don't forget to update your code.